migration/migration-integrators.md

```diff
@@ -71,9 +36,8 @@ The following methods are particularly useful when interacting with the network:
 7. [`createBasicTransaction`](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#createbasictransaction): Create a signed transaction and returns a hex-encoded representation without broadcasting it. All `create*Transaction` RPC methods also have an equivalent `send*Transaction` version in order to create and broadcast the transaction one-go.
 8. [`sendRawTransaction`](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#sendrawtransaction): Sends the given serialized and signed transaction to the network.
 
-### Shutdown of PoW Nodes
-
-Once the PoS chain has started, PoW nodes are no longer needed and can be shut down. This also acts as a commitment to leave the PoW chain behind. Make sure that accessing transactions on the PoW chain is no longer necessary once the PoS Mainnet chain is live. **If you need to access transaction history from before the transition, you will need to complete the migration process as described in the [Migration Tool Guide](node-operators.md).**
+#### Legacy Data Access
+The ability to fetch transactions from before the PoS genesis (PoW transactions) will be documented in a future update. These transactions use a different format compared to PoS transactions. Instructions on how to handle and query these pregenesis transactions will be provided soon.
```

Comment:
Here I would add a small alinea explaining that Nimiq has migrated from PoW to PoS by now. And that there is a read-only database of the PoW chain available if it is desired to be able to fetch information prior to the PoS genesis block.